src/__init__.py

```diff
@@ -386,6 +390,15 @@ def client_data(self):
             'regions': region_table,
             'categories': category_table
         }
+    
+    # for the universal tracker, doesn't get called in standard gen
+    @staticmethod
+    def return_slot_data(slot_data):
+        # returning slot_data so it regens, giving it back in multiworld.re_gen_passthrough
+        return slot_data
+    
+    if ut_enabled:
+        interpret_slot_data = return_slot_data
```

Comment:
It's worth noting that we already have a `interpret_slot_data` at https://github.com/ManualForArchipelago/Manual/pull/34/files#diff-6b39a2f1a6399aa39e427fb4c11a7b552e615a2d83a3c36aa6dd6ddbbb8281f4R82-R84 and a hook for it in hooks/data.py (Why is it there??)

Also, this somehow conflicts with #35

Reply:
Ok I fixed this to use the correct interpret_slot_data function

I also removed the weird hook that doesn't do anything